packages/slice-machine/tests/server/env.test.js

```diff
@@ -229,20 +236,21 @@ describe("getEnv", () => {
     expect(env.framework).toEqual("vanillajs");
   });
 
-  test("it should take the auth from .prismic and base from sm.json", async () => {
+  test("it's client should use autentication token from .prismic", async () => {
```

Comment:
user's probably won't run into the situation where the toke is not for prismic.io

Reply:
nope you're right, but we might haha.